app/commands/exercise/representation/find_example_submissions.rb

```diff
@@ -0,0 +1,44 @@
+class Exercise::Representation::FindExampleSubmissions
+  include Mandate
+
+  initialize_with :representation
+
+  def call
+    example_submissions = [representation.source_submission]
+    example_submission_hashes = Set.new([solution_files_hash(representation.source_submission)])
```

Comment:
Alternatively we could have one hash of `hash: submission` then check for the `key?` below, then end the method by returning `values`. Rather than keeping two lists in play.